Tags added in Description

SharathGa opened the pull request:
Attempts to fix #44
`setup.py`  was modified and few tags were added in the Description. Tags are related to neural networks.

SharathGa committed:
Description modified

added few tags related to neural networks

SharathGa committed:
update description

Chronocook committed:
Merge branch 'master' into desc

Chronocook commented:
This one needs a tweak, setup.py doesn't like multiple strings under `description` 
the description field should be a short string as per https://docs.python.org/2/distutils/setupscript.html 
We can add a `long_description` too

SharathGa committed:
Added long_description

new parameter long_description was added so that the package description
is verbose and improves the visibility of the project.

SharathGa committed:
Merge branch 'desc' of https://github.com/SharathGa/numpynet into desc

Chronocook reviewed (approved):
Sorry this is so late, i took a short break from open source stuff.
This looks good i'll try to merge it, thanks :)

Chronocook commented:
setup.py doesn't like having lists in it... let me look into the long_description field and i'll fix it
